src/styles/community.scss

```diff
@@ -30,6 +30,7 @@
   }
 
   &.side-nav__item-community--done::before {
+    /* stylelint-disable scss/operator-no-unspaced */
```

Comment:
Think it's better to just quote the data url instead of escaping, but don't feel that strongly